src/frontend/src/lib/components/ui/WarningBanner.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import IconWarning from '$lib/components/icons/IconWarning.svelte';
+</script>
+
+<div
+	class="mx-auto inline-flex w-full items-center justify-center gap-2 px-6 py-2 text-xs font-bold sm:w-fit md:text-base text-american-orange bg-cornsilk border border-crayola-yellow rounded-lg"
```

Comment:
i see you left `mx-auto` in the hyperlink (makes sense), but you put it here too: is that deliberately?

Reply:
Let me remove it, it's a good point. Not sure it's needed anymore.

Reply:
Done